__tests__/crawler.test.js

```diff
@@ -129,6 +129,7 @@ test("Crawl the docs and execute tests", async () => {
     'https://calicousers.slack.com/',
     'https://calicousers.slack.com/archives/C017220EXU1',
     'https://tigera-manager.mycompany.com',
+    'https://kb.isc.org/article/AA-01141/31/How-to-workaround-IPv6-prefix-length-issues-with-ISC-DHCP-clients.html',
```

Comment:
Does this need to be fixed in source or should we add it here?

Reply:
Yes, that is my intention. Just testing now to see if the other dead link remains. This is the PR I want to discuss on zoom.